src/tests/scripts/lca_test_environment.sh

```
@@ -0,0 +1,47 @@
+gunzip --quiet --force ${TMP_DIR}/$KBGZ
+das-cli db stop --prune >> ${TMP_DIR}/atomdb.log 2>&1
+(docker stop `docker ps | tail -n +2 | cut -d" " -f1` ; docker container prune -f ; docker ps -a) > /dev/null 2>&1
+das-cli db start >> ${TMP_DIR}/atomdb.log 2>&1
+make run-db-loader OPTIONS="--config=/opt/das/config/das.json --file=${TMP_DIR}/$KB --threads=4 --chunk=5000" >> ${TMP_DIR}/atomdb.log 2>&1
+make run-attention-broker >> ${TMP_DIR}/attention_broker.log 2>&1 &
+sleep 5
+make run-busnode OPTIONS="--service=query-engine --config=/opt/das/config/das.json" >> ${TMP_DIR}/query_engine.log 2>&1 &
+sleep 5
+make run-busnode OPTIONS="--service=link-creation-agent --bus-endpoint=localhost:40002 --config=/opt/das/config/das.json" >> ${TMP_DIR}/link_creation_agent.log 2>&1 &
+sleep 5
+\rm -rf ${TMP_DIR}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)